fal/controllers/load_aces.py

```diff
@@ -68,7 +68,7 @@ def load_aces(input_lines: Iterable[str]) -> None:
         for line in input_lines:
             teamname, animename = line.split(" ", 1)
             team = Team.get_team_from_database(teamname, season, session)
-            anime = Anime.get_anime_from_database_by_name(animename, session)
+            anime = Anime.get_anime_from_database_by_name(animename.strip(), session)
```

Comment:
I just realized that I forgot to strip the title in `get_licensed_anime()` in `anime_stats.py`. Do you mind adding that here: https://github.com/fantasy-anime-league/fantasy-anime-league/blob/master/fal/controllers/anime_stats.py#L196

I'm gonna approve so you don't have to ask me to review again

Reply:
thanks for catching that as well